Framework/Algorithms/src/Rebin2D.cpp

```diff
@@ -126,22 +128,23 @@ void Rebin2D::exec() {
   {
     PARALLEL_START_INTERUPT_REGION
 
+    m_progress->report("Computing polygon intersections");
```

Comment:
Great improvement 👍 .
I guess the bar still updates at a decent rate with you latter suggestion? Then I'd go for that.